nex/nex.go

```diff
@@ -80,6 +80,7 @@ func init() {
 	run.Flag("name", "Name of the workload. Must be alphabetic (lowercase)").Required().StringVar(&RunOpts.Name)
 	run.Flag("type", "Type of workload, e.g., \"elf\", \"v8\", \"oci\", \"wasm\"").StringVar(&RunOpts.WorkloadType)
 	run.Flag("description", "Description of the workload").StringVar(&RunOpts.Description)
+	run.Flag("argv", "Arguments to pass to the workload, if applicable").StringVar(&RunOpts.Argv)
```

Comment:
Will need this in `yeet` too

Reply:
I realize my current handling of argv is naive at best but it should get us started.